Localization/StringsConvertor/input/pt-BR.lproj/ios-infoPlist.json

```diff
@@ -1,6 +1,6 @@
 {
-    "NSCameraUsageDescription": "Used to take photo for post status",
-    "NSPhotoLibraryAddUsageDescription": "Used to save photo into the Photo Library",
-    "NewPostShortcutItemTitle": "New Post",
-    "SearchShortcutItemTitle": "Search"
+    "NSCameraUsageDescription": "Usado para tirar uma foto para postagem",
+    "NSPhotoLibraryAddUsageDescription": "Usado para salvar foto na Galeria",
+    "NewPostShortcutItemTitle": "Novo Toot",
```

Comment:
Here the work `Toot` is still used, should probably be swapped by the `Post`equivalent?